### ISSUE TITLE
Compatibility with gtk-rs 0.6

### ISSUE THREAD

**Issue by jangernert** [gtk-rs/webkit2gtk-rs#47]
```
failed to select a version for `gdk-sys`.
    ... required by package `webkit2gtk v0.5.0`
versions that meet the requirements `^0.7.0` are: 0.7.0

the package `gdk-sys` links to the native library `gdk`, but it conflicts with a previous package which links to `gdk` as well:
package `gdk-sys v0.8.0`
    ... which is depended on by `gtk-sys v0.8.0`
    ... which is depended on by `gtk v0.6.0`

failed to select a version for `gdk-sys` which could resolve this conflict
```

So you basically have to use gtk-rs 0.5 to be able to use webkit2gtk-rs right now.

**Comment by antoyo** [comment]
I guess it should be done on master. We just need to update the crate branch.

**Comment by antoyo** [comment]
Fixed by #50.
